src/js/utils/backbone.events.ts

```diff
@@ -27,7 +27,7 @@ type EventListener = {
 }
 
 type EventCallback = {
-    (): void;
+    (...args: any): void;
```

Comment:
I think this has to be an array type like `any[]`

Reply:
great catch